### PR TITLE
Add Better Titles

### PR DESCRIPTION
**Pull request by aschmois**
Updating mods was really a hard thing to do when I had many tabs open
and couldn't find which was which. This simple change made it multiple
times better to organize my tabs.

Changing the title of the view can now be done by just adding the
section "title" to the page and adding the title tag inside of it.

### CODE REVIEW

**Comment by Isigiel** [comment]
That looks quite good ...

File: app/views/mod/view.blade.php [hunk 1]
```diff
@@ -1,4 +1,7 @@
 @extends('layouts/master')
+@section('title')
+    <title>{{ $mod->pretty_name }} - TechnicSolder</title>
```
Comment (Pyker) [inline app/views/mod/view.blade.php]: Perhaps `{{ empty($mod->pretty_name) ? $mod->name : $mod->pretty_name }}` here, like lines 11-15?

Comment (aschmois) [inline app/views/mod/view.blade.php]: I don't see why not, I'll add it.

**Comment by Pyker** [comment]
Looks good! :+1:

**Comment by aschmois** [comment]
I think Travis drank too much, there's no reason that should fail. I don't see empty being deprecated.

**Comment by Pyker** [comment]
The Travis failure is unrelated to your changes.

**Comment by GenPage** [comment]
Please resubmit this request on the dev branch

**Pull request closed without merging**